Gems/ScriptCanvas/Code/Source/ScriptCanvasGem.cpp

```diff
@@ -7,7 +7,7 @@
  */
 
 
-#if !defined(SCRIPTCANVAS_EDITOR)
+#if defined(SCRIPTCANVAS_EDITOR)
```

Comment:
This causes double registration of the ScriptCanvas common System Component Descriptors in the Tooling application such as the Editor and Asset Processor.

It also prevents ScriptCanvas from working at all in the launcher.

Reply:
ScriptCanvas should not work in the launcher, ever since it generates Lua code, there is no runtime component to ScriptCanvas that needs to run

Reply:
How about interpreted mode?
The ScriptCanvas::SystemComponent isn't created anymore with this change.

Are you sure that the translated ScriptCanvas graph to lua still works?

Reply:
That's the only way Script Canvas works